app/(pages)/orders/view-order/_components/ShippingLabel/ShippingLabel.tsx

```diff
@@ -0,0 +1,41 @@
+'use client';
+
+import Image from 'next/image';
+import { useState } from 'react';
+
+import styles from './ShippingLabel.module.scss';
+
+export default function ShippingLabel() {
+  const [showLabel, setShowLabel] = useState(false);
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.header_container}>
+        <div className={styles.header}>
+          <Image
+            src="/icons/label.svg"
+            alt="label icon"
+            width={20}
+            height={20}
+          />
+          Shipping Label
+        </div>
+        <button onClick={() => setShowLabel(!showLabel)}>
+          {showLabel ? 'See Less' : 'See More'}
+        </button>
+      </div>
+      {showLabel && (
+        <div className={styles.img_container}>
+          <Image
```

Comment:
Read up on static image rendering. https://nextjs.org/docs/app/building-your-application/optimizing/images#local-images

Dynamic images are fine in this setting though since in the future, these images will be pulled from cloudinary.